frontend/taipy-gui/src/components/Taipy/Button.tsx

```diff
@@ -40,7 +44,8 @@ const Button = (props: ButtonProps) => {
     const className = useClassNames(props.libClassName, props.dynamicClassName, props.className);
     const active = useDynamicProperty(props.active, props.defaultActive, true);
     const hover = useDynamicProperty(props.hoverText, props.defaultHoverText, undefined);
-
+    const variant = useDynamicProperty(props.variant,props.defaultVariant,"outlined") as "outlined"| "text" | "contained";
+    const size = useDynamicProperty(props.size,props.defaultSize,"medium") as "small"| "medium" | "large";
```

Comment:
no need to make these dynamic

Reply:
Hi @FredLL-Avaiga, I couldn’t find any hooks that return a string type. Does this approach is valid here?
const variant = (props.variant ?? "outlined") as "outlined" | "text" | "contained"

Reply:
No need for hooks
You can simply set the default value when you destructure the props

Reply:
Please change it
You can set the default value in the line destructuring the props
Something like
const { variant = "outlined", ... } = props;

Reply:
Done, I’ve changed it. Thanks!